app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -429,7 +429,7 @@ class ProfileResetPinActivityTest {
   }
 
   @Test
-  fun testResetPin_asAdmin_inputWrongConfirmPin_clickSave_changeConfig_checkConfirmWrongError() {
+  fun testProfileResetPin_withAdmin_wrongConfirmPin_save_changeConfig_checkConfirmWrongError() {
```

Comment:
testProfileResetPin_withAdmin_inputWrongConfirmPin_changeConfig_confirmWrongErrorIsDisplayed

let me know if it works!!

Reply:
(Sorry for replying to the wrong comment)

Exceeded the limit, instead changed confirmWrongErrorIsDisplayed to confirmWrongErrorIsShown throughout the file, similarly for other functions.